src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Tree/Dependencies/Subscriptions/DependencyRulesSubscriberBase.cs

```diff
@@ -81,14 +75,7 @@ public void ReleaseSubscriptions()
 
             _subscriptions ??= new DisposableBag();
```

Comment:
maybe we no longer need a DisposableBag here as we have only one link to dispose.